src/custom-element.js

```diff
@@ -968,6 +1158,10 @@ function createBaseCustomElementClass(win) {
       if (this.impl_) {
         this.impl_.detachedCallback();
       }
+      if (!this.built_ && this.V2()) {
```

Comment:
FMI: The builder would never schedule a disconnected element anyway right? (since it is InOb based). I'm guessing this is just to free up a tiny bit of mem

Reply:
Possibly not that tiny. The targets are kept in the hard map, so I don't want to introduce a hard memory leak here.